nifi-docs/src/main/asciidoc/toolkit-guide.adoc

```diff
@@ -1111,6 +1109,9 @@ coefficient:
     7a:00:bb:07:e9:13:91:82:9b
 ```
 
+[[additional-commands]]
+==== Additional Commands
```

Comment:
Change text to "Additional Certificate Commands" to be consistent with the text from the Admin Guide link.

Also, is this section within the Intermediate CA section or outside?  If outside, change to:

=== Additional Certificate Commands

Reply:
Done.